installer/dockerfile/controller-manager/Dockerfile

```diff
@@ -12,8 +12,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+FROM golang:1.17 AS builder
+WORKDIR /go/src/volcano.sh/
+ADD . volcano
+RUN cd volcano && make vc-controller-manager
 
 FROM alpine:latest
 
-ADD vc-controller-manager /vc-controller-manager
+COPY --from=builder /go/src/volcano.sh/volcano/_output/bin/vc-controller-manager /vc-controller-manager
```

Comment:
```suggestion
ARG TARGETPLATFORM
COPY --from=builder /go/src/volcano.sh/volcano/_output/bin/$TARGETPLATFORM/vc-controller-manager /vc-controller-manager
```

Reply:
@hwdef could you also help to confirm, does it works in windows?

Reply:
> @hwdef could you also help to confirm, does it works in windows?

ok, I'll check this in windows,but it may have to wait until after the Qingming Festival, because my windows environment is in the company.